fdbserver/FDBExecHelper.actor.cpp

```diff
@@ -84,13 +84,13 @@ void ExecCmdValueString::dbgPrint() {
 }
 
 #if defined(_WIN32) || defined(__APPLE__)
-ACTOR Future<int> spawnProcess(std::string binPath, std::vector<std::string> paramList, double maxWaitTime, bool isSync)
+ACTOR Future<int> spawnProcess(std::string binPath, std::vector<std::string> paramList, double maxWaitTime, bool isSync, double maxSimDelayTime)
```

Comment:
I think this would be more readable if you wouldn't repeat the definition of the actor in each block of `#if` and instead only guard the inner code with preprocessor definitions. This would also be more consistent with what the rest of fdb code is looking like (see for example `flow/Platform.cpp` where this pattern is used all over the place). It would also allow you to change the signature only once if you need to change it (otherwise this will potentially introduce compiler errors on platforms you don't explicitely test on)

Reply:
Yes, that was my first attempt. Unfortunately, actors are not friendly with #if blocks in it, compilation will break. Examples in flow/Platform.cpp and other places are regular functions and not ACTORs.

Reply:
Ah I see, this actually makes sense...